recipes/rectanglebinpack/all/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(NOT "${CMAKE_CXX_STANDARD}")
+    set(CMAKE_CXX_STANDARD 11)
+endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
I will submit this to upstream, but since the upstream cmake file has two targets (`RectangleBinPack` and `MaxRectsBinPackTest`) which need C++11 support, isn't it better to just define `set(CMAKE_CXX_STANDARD 11)` instead of defining `target_compile_features` for both targets?

Reply:
No, `CMAKE_CXX_STANDARD` should never be used in libraries. Nowadays `target_compile_features` should be preferred in libraries, it allows to ask for a minimum C++ standard (while `CMAKE_CXX_STANDARD` asks for the upper bound).
Then final project can decide to enforce a specific C++ standard through `CMAKE_CXX_STANDARD` in all its dependencies.

It also plays better with `compiler.cppstd` in conan profiles.

Reply:
All right, I'll use `target_compile_features`, thanks for the explanation!